WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -98,11 +170,21 @@ private extension CardReaderSettingsConnectedViewController {
         cell.selectionStyle = .none
     }
 
-    private func configureButton(cell: ButtonTableViewCell) {
-        cell.configure(title: Localization.buttonTitle) { [weak self] in
+    private func configureUpdateButton(cell: ButtonTableViewCell) {
+        cell.configure(style: .primary, title: Localization.updateButtonTitle, bottomSpacing: 0) {
+            // TODO in a following PR
+        }
+        cell.selectionStyle = .none
+        cell.backgroundColor = .clear
+    }
+
+    private func configureDisconnectButton(cell: ButtonTableViewCell) {
+        let style: ButtonTableViewCell.Style = shouldShowUpdateControls() ? .secondary : .primary
+        cell.configure(style: style, title: Localization.disconnectButtonTitle) { [weak self] in
             self?.viewModel?.disconnectReader()
         }
         cell.selectionStyle = .none
+        cell.backgroundColor = .clear
```

Comment:
This was missed in previous PRs. This updates the button background to match the designs in Figma.